### ISSUE TITLE
examples question and getting started

### ISSUE THREAD

**Issue by philr35** [joshuaferrara/node-csgo#52]
Hi , so I want to get started with this repo to make a website for csgo that pulls certain information from the game/player.

I'm having a hard time getting started. While the documentation on the readme is superb, there's isn't enough for someone new to get started.

I saw there was an example.js, but none of it is commented, and its using a lot of stuff not mentioned in the readme. Even when I run the example.js, after setting my username and password, it asks for an auth code, and no where in there does it specify how to get an auth code.

Basically, if I'm a web developer and all I want to do is pull off some user data what's the best way to do that, Can someone provide me with a full code snippet on something that works, so I can get started? I couldn't find any tutorials online for this specific repo.

Please don't get mad if this seems like a dumb post, I'm just trying to get started :)

**Comment by joshuaferrara** [comment]
The auth code will be the code steam emails you for SteamGuard. If you have
two factor enabled, I can't help you much there. I've heard there is some
success using the node-steam-user package to work with 2FA. You can also
ask in Gitter channel; currently busy at the moment but they should be able
to provide you with more examples.

**Comment by philr35** [comment]
@joshuaferrara thanks for your quick response. I'm trying my best to figure this out.

So quick question. I can use any account to login, and if I have another players steam id I can get info on them correct? Using the various different functions in this repo.

**Comment by joshuaferrara** [comment]
Correct, provided the account you are logging in with has CS:GO purchased,
AND is friends with the account you are querying information on.

**Comment by philr35** [comment]
So the website I want to create is sort of a match making website for competitive play using react and express. If you say, only if they're friends with the account. How do websites like csgo-mates look up people in games? Obviously, they're not friends with the account they're using.

**Comment by philr35** [comment]
@joshuaferrara I think I've figured out the above, and what exactly I'll need.

One last question and I think I'll be good to go! So, currently, to get the auth code for the example application, I have to manually try to login in my desktop which triggers an auth code being sent to my email. It works once I put the same auth code into the example application. Works perfectly

But how would I trigger the auth code being sent to my email without manually having to login on my desktop? Is there a way to do that? Also, is it a one time login, meaning if the server is up 24/7 I won't need to do it again?

**Comment by joshuaferrara** [comment]
If you attempt to login without an authcode, or with an empty one, using `node-steam`, you should receive an email with an authcode. 

If you're using the example verbatim, meaning with the `readline-sync` module, just press enter when you are asked for the authcode, and you'll receive an email with it.

**Comment by philr35** [comment]
@joshuaferrara what's the best way to get the auth code from my email into node? I found this module [Nodemailer](https://nodemailer.com/about/), but it says for sending only. I cant find one that is for opening emails so I can get the code.

Should I not bother, and just input it manually?

**Comment by joshuaferrara** [comment]
You should only need to get the code once. The example will create a `sentry` file, which will allow you to login without auth codes in the future.
